pkg/controller/node/nodecontroller.go

```diff
@@ -55,6 +55,7 @@ import (
 	utilversion "k8s.io/kubernetes/pkg/util/version"
 
 	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/kubelet"
```

Comment:
Agree, so went ahead and created #45178 ... would you prefer that as pre-requisite to this ? Im ok either way.